gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -752,6 +758,30 @@ case class ColumnarOverrideRules(session: SparkSession)
     fallbacks >= wholeStageFallbackThreshold
   }
 
+  def fallbackWholeQuery(plan: SparkPlan): Boolean = {
```

Comment:
the logical of this function is the same as fallbackWholeStage, can it be removed ?

Reply:
indeed, will merged in to one function